helpers/vec.py

```diff
@@ -0,0 +1,54 @@
+import math
+
+# https://gist.github.com/tatsy/e14dd18079bca60ac8f78217b77332c1
+
+class Vec(object):
```

Comment:
I already added something similar to this (a Vec2) in engine/mathlib (though currently unused). They should probably be in the same place at least.